src/EditorFeatures/Core/LanguageServer/EditorLspReferencesResultCreationService.cs

```diff
@@ -61,11 +57,13 @@ public EditorLspReferencesResultCreationService()
             result.ProjectName = documentSpan.Value.Document.Project.Name;
         }
 
-        if (properties.TryGetValue(AbstractReferenceFinder.ContainingMemberInfoPropertyName, out var referenceContainingMember))
-            result.ContainingMember = referenceContainingMember;
-
-        if (properties.TryGetValue(AbstractReferenceFinder.ContainingTypeInfoPropertyName, out var referenceContainingType))
-            result.ContainingType = referenceContainingType;
+        foreach (var (key, value) in properties)
+        {
+            if (key == AbstractReferenceFinder.ContainingMemberInfoPropertyName)
+                result.ContainingMember = value;
+            else if (key == AbstractReferenceFinder.ContainingTypeInfoPropertyName)
```

Comment:
>else [](http://example.com/codeflow?start=12&length=4)

Just making sure these are mutually exclusive

Reply:
yes.  these are constants, set to different values.